src/sink.rs

```diff
@@ -1,15 +1,14 @@
 use crate::stream::{OutputStreamHandle, PlayError};
-use std::sync::atomic::Ordering;
-use std::sync::atomic::{AtomicBool, AtomicUsize};
-use std::sync::mpsc::Receiver;
-use std::sync::Arc;
-use std::sync::Mutex;
-use std::time::Duration;
-
-use crate::queue;
-use crate::source::Done;
-use crate::Sample;
-use crate::Source;
+use std::{
```

Comment:
I don't like this change, can you keep it? Only the `std::sync::atomic::Ordering` should be put into the group of `{AtomicBool, AtomicUsize, Ordering}`.